Firestore/core/src/firebase/firestore/auth/token.h

```diff
@@ -44,7 +44,7 @@ class Token {
  public:
   Token(const absl::string_view token, const User& user);
 
-  /** The actual raw token. */
+  /** The actual raw token; will be "" if the user is unauthenticated. */
```

Comment:
This doesn't seem right. A valid token is not "". An invalid token might be "" but this should have asserted before.